pymemcache/pool.py

```diff
@@ -63,19 +67,26 @@ def get_and_release(self, destroy_on_fail=False):
 
     def get(self):
         with self._lock:
-            if not self._free_objs:
+            # Find a free object, removing any that have idled for too long.
+            now = self._idle_clock()
+            while self._free_objs:
+                obj = self._free_objs.popleft()
+                if now - obj._last_used > self.idle_timeout:
+                    self._after_remove(obj)
+                else:
+                    break
```

Comment:
Aesthetically, I would prefer the loop to be:

```
if now - obj._last_used <= self.idle_timeout:
    break

if self._after_remove is not None:
    self._after_remove(obj)
```

Reply:
Done.